Quadriga/src/main/java/edu/asu/spring/quadriga/aspects/ConceptCollectionRestAuthorization.java

```diff
@@ -30,26 +33,27 @@
     @Autowired
     private IConceptCollectionFactory collectionFactory;
 
-    /**
-     * This method checks the access permission for the logged in user for given
-     * concept collection for rest methods.
-     * 
-     * @param userName
-     *            - username submitted for rest authentication.
-     * @param conceptCollectionId
-     *            - Concept collection id.
-     * @param userRoles
-     *            - Roles for which the user should be checked for access.
-     * @return - if has access - true no access - false
-     * @author Lohith Dwaraka.
-     */
+    private final Logger logger = LoggerFactory.getLogger(ConceptCollectionRestAuthorization.class);
+
     @Override
-    public boolean chkAuthorization(String userName,
-            String conceptCollectionId, String[] userRoles)
+    public boolean chkAuthorization(String userName, Object conceptCollectionObj, String[] userRoles)
             throws QuadrigaStorageException, QuadrigaAccessException {
 
         // fetch the details of the concept collection
-        IConceptCollection collection = conceptCollectionManager.getConceptCollection(conceptCollectionId);
+        IConceptCollection collection;
+        String conceptCollectionId = null;
```

Comment:
same as above. please pay more attention to the flow and when what variable is used an set.